android/assets/jsons/translationsByLanguage/Traditional_Chinese.properties

```diff
@@ -535,11 +536,11 @@ Receive triple Gold from Barbarian encampments and pillaging Cities. Embarked un
 
 Spain = 西班牙
 Isabella = 伊莎貝拉一世
-100 Gold for discovering a Natural Wonder (bonus enhanced to 500 Gold if first to discover it). Culture, Happiness and tile yields from Natural Wonders doubled. = 每發現一座自然奇觀時獲得100金錢獎勵(若為第一個發現該自然奇觀的文明，獎勵則追加至500)。自然奇觀所提供的文化、快樂及地區產出雙倍。
+100 Gold for discovering a Natural Wonder (bonus enhanced to 500 Gold if first to discover it). Culture, Happiness and tile yields from Natural Wonders doubled. = 每發現一座自然奇觀時獲得100金錢獎勵(若為第一個發現該自然奇觀的文明，獎勵則追加至500)，自然奇觀所提供的文化、快樂及地區產出雙倍
 
 Mongolia = 蒙古
 Genghis Khan = 成吉思汗
-Combat Strength +30% when fighting City-State units or attacking a City-State itself. All mounted units have +1 Movement. = 進攻城邦及城邦單位時獲得 30% 戰鬥力加成。騎乘單位移動力+1。
+Combat Strength +30% when fighting City-State units or attacking a City-State itself. All mounted units have +1 Movement. = 進攻城邦及城邦單位時獲得30%戰鬥力加成，騎乘單位移動力+1
```

Comment:
Why do you change this translation? It has nothing to the Effel Tower effect.

Reply:
The same question - to all other altered translations in this file.

Reply:
some translation update.